x/emissions/keeper/params_test.go

```diff
@@ -0,0 +1,213 @@
+package keeper_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
Would be better to only use require for consistency

Reply:
Replacing require with asser in this pr 
https://github.com/zeta-chain/node/pull/1694

let's get this one merged though,if everything else looks fine